storm/src/main/storm/mesos/resources/ResourceEntries.java

```diff
@@ -93,15 +93,15 @@ public ReservationType getReservationType() {
     public RangeResourceEntry add(ResourceEntry<Long> resourceEntry) {
```

Comment:
My question was more specific.  As I noted, I get the gist of what you're doing, I just cannot figure out *precisely* how you are trying to handle ranges.